domain/domain.go

```diff
@@ -1575,6 +1580,46 @@ func (do *Domain) SetStatsUpdating(val bool) {
 	}
 }
 
+// ReturnAnalyzeExtraExec returned extra exec for Analyze
+func (do *Domain) ReturnAnalyzeExtraExec(sctxs []sessionctx.Context) {
```

Comment:
It seems the function do the reset operation. But the function name is `Return...`. It's a little bit confusing.

Reply:
Oh, I get the meaning for `return` from the following context. But I think it's better to change the name to understand easier.